[cra-rxjs-sc] update fetch example loader to match other kits

sheerikie opened the pull request:
# What was done?
- Removed Tailwind,autoprefixer,postcss
- Replaced loader on Fetch Example with new Loader component
- Fixed .gitignore whitespaces

# Screenshot/Video/GIF of Changes (if applicable)
TODO: @sheerikie 
[Screencast from 03-08-2022  8:55:27 WB.webm](https://user-images.githubusercontent.com/4911645/182682235-6e872233-92ee-44a7-8017-e7ac2429d880.webm)


# Related Ticket(s)
Resolves #206

sheerikie commented:
refactor rxjs

dustinsgoodman commented on starters/cra-rxjs-styled-components/.gitignore:
```diff
@@ -21,3 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+
```
Let's avoid the extra whitespace

dustinsgoodman commented on starters/cra-rxjs-styled-components/package.json:
```diff
@@ -9,12 +9,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "autoprefixer": "^10.4.7",
+    "postcss": "^8.4.14",
```
We should not be installing tailwind. This project is intended to only use Styled Components. I linked the original Tailwind article as a guide for what CSS you could use to make this show up quickly since Tailwind translates to plain CSS pretty easily.

dustinsgoodman commented on starters/cra-rxjs-styled-components/src/components/FetchExample/FetchExample.tsx:
```diff
@@ -1,35 +1,52 @@
-        <ReturnHomeLink to="/">Return Home</ReturnHomeLink>
-      </HomeLinkDiv>
-    </FetchExampleContainer>
+  const stream$ = useMemo(
+    () =>
+      fromFetch(
+        'https://api.starter.dev/hello?greeting=from This Dot Labs!'
+      ).pipe(
+        mergeMap((response) => response.text()),
+        // now we'll map not only to text
+        // but to JSX
+        map((data) => (
+          <FetchExampleContainer>
+            <HeaderContainer>
+              <Header>RxJS Fetch Data from API</Header>
+            </HeaderContainer>
+            <Message>Message: {data}</Message>
+            <HomeLinkDiv>
+              <ReturnHomeLink to="/">Return Home</ReturnHomeLink>
+            </HomeLinkDiv>
+          </FetchExampleContainer>
+        )),
+        catchError(() => of(<div className="err">ERROR</div>)),
+        startWith(<Loader />)
+      ),
+    []
   );
+
+  return <$>{stream$}</$>;
```
The previous component was correct. We want to just replace the `<Message>` with the `Loader`, not the entire component.

dustinsgoodman commented on starters/cra-rxjs-styled-components/src/index.css:
```diff
@@ -1,13 +0,0 @@
-body {
```
Don't delete this - it ruins the nice font we had.

sheerikie commented:
# What was done?
- Added Tailwind
- Created Loader component
- Replaced loader on Fetch Example with new Loader component

# Screenshot/Video/GIF of Changes (if applicable)
TODO: @sheerikie 
[Screencast from 03-08-2022  8:55:27 WB.webm](https://user-images.githubusercontent.com/4911645/182682235-6e872233-92ee-44a7-8017-e7ac2429d880.webm)


# Related Ticket(s)
Resolves #206
